js/core/webidl-contiguous.js

```diff
@@ -640,9 +640,12 @@ define(
 
                     case "enum":
                         name = defn.name;
-                        defn.values.filter(function (v) { return v.type === undefined;})
-                                   .forEach(function(v) {
-                            v.dfn = findDfn(name, v, definitionMap, msg);
+                        defn.values.forEach(function(v,i) {
+                            if (v.type === undefined) {
```

Comment:
(if I got the above wrong, and we just need a hot fix, then quickly then just merge ... this bit is not particularly complicated in the overall scheme of things)